components/net/indexeddb/thread_pool.rs

```diff
@@ -0,0 +1,53 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+use crossbeam_channel::{self, Receiver, Sender};
+use std::thread;
+
+#[derive(Clone)]
+pub struct TransactionPool {
```

Comment:
For this you can consider using a pool from `rayon`, fetch for example uses one at https://github.com/servo/servo/blob/4d5bf653f72a4ebf02c627eda7c5fa7f3744cbe8/components/net/resource_thread.rs#L496